lib/galaxy/managers/workflows.py

```diff
@@ -90,9 +90,14 @@ def check_security(self, trans, has_workflow, check_ownership=True, check_access
         if not check_ownership or check_accessible:
```

Comment:
Also `get_owned_workflow` seems pretty strict as a requirement. If we fix this it would mean a shared workflow that you don't own can't be part of a subworkflow.